firstvoices/backend/urls.py

```diff
@@ -1,24 +1,20 @@
 from django.urls import path
 from rest_framework.routers import DefaultRouter
-from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView
 
-from backend.views.user import UserViewSet
-from backend.views.sites import SiteViewSet
+from backend.views.dictionary_views import PartsOfSpeechViewSet
 from backend.views.health import HealthCheckView
-
+from backend.views.sites_views import SiteViewSet
+from backend.views.user import UserViewSet
 
 ROUTER = DefaultRouter(trailing_slash=False)
-ROUTER.register(r'api/1.0/user', UserViewSet, basename=r'user')
-ROUTER.register(r'api/1.0/sites', SiteViewSet)
+ROUTER.register(r"user", UserViewSet, basename=r"user")
+ROUTER.register(r"sites", SiteViewSet, basename="site")
+ROUTER.register(r"parts-of-speech", PartsOfSpeechViewSet, basename="parts-of-speech")
+
+app_name = "api"
 
 urlpatterns = [
-	path(r'health', HealthCheckView.as_view()),
-	# path("api/schema/", SpectacularAPIView.as_view(), name="api-schema"),
-	# path(
-	# 	"api/docs/",
-	# 	SpectacularSwaggerView.as_view(url_name="api-schema"),
-	# 	name="api-docs",
-	# ),
+    path(r"health", HealthCheckView.as_view()),
```

Comment:
I realize this changes the url of the health check. My proposal is to deal with that as its own issue AFTER the merge